src/conn/conn_open.c

```diff
@@ -139,6 +145,9 @@ __wti_connection_close(WT_CONNECTION_IMPL *conn)
     /* Disconnect from shared cache - must be before cache destroy. */
     WT_TRET(__wti_conn_cache_pool_destroy(session));
 
+    /* Destroy Eviction. */
+    WT_TRET(__wti_evict_destroy(session));
```

Comment:
We have the same naming conflict here. I suggest renaming `__wt_evict_destroy` to `__wt_evict_server_destroy`

Reply:
I am planning to leave `__wt_evict_destroy` and re-naming the current `__wt_evict_destroy` -> `__wt_evict_threads_destroy`.